build-tools/heroku/tailwind/styles.css

```diff
@@ -536,3 +536,19 @@ input:checked ~ .toggle-path {
 .current {
   @apply text-white bg-blue-600 !important;
 }
+
+.animate-keys {
+  position: absolute;
+  bottom: 15px;
+  right: 15px;
+
+  animation-name: animate-keys;
+  animation-duration: 1.5s;
+  animation-timing-function: ease-in-out;
+  -webkit-animation-timing-function: ease-in-out;
+}
+
+@keyframes animate-keys {
+  0%   {bottom:15px; font-size: 1em; opacity: 1}
+  100% {bottom:100%; font-size: 4em; opacity: 0}
+}
```

Comment:
One small suggestion already: It would be nice to make this class dependent on Tailwind as well. Don't worry, I can take a look at this to change into a tailwind dependency.

Reply:
I have never used Tailwind😅, so I'm not so sure on how to implement it in that way. I would be happy to see how you do that!